### ISSUE TITLE
mfhelper doesnt tp leader

### ISSUE THREAD

**Issue by boedh** [kolton/d2bot-with-kolbot#1012]
both leader and helper joins game but then it goes wrong. Helper doesnt w8 for tp, goes to waypoint and tries to reach chaos himselff. What im i missing? tnx fot the help in advance:-)

**Comment by JohnGoblikon** [comment]
Sounds like helper needs river wp.

**Comment by boedh** [comment]
yeah. indeed. I saw the light and gaved it to him and finally got it to work:-)
